src/components/ModalCreateMessage.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+  <div class="wrapper">
+    <div class="input-box">
+      <div class="input-label">Title</div>
```

Comment:
Any reason for not using a label tag here?
```suggestion
      <label class="input-label">Title</label>
```

Reply:
Fixed. It was not supposed to be a label before, so then I've changed it and forgot to update the element.